.github/CODEOWNERS

```diff
@@ -28,7 +28,7 @@ confmap/provider/s3provider/                             @open-telemetry/collect
 connector/countconnector/                                @open-telemetry/collector-contrib-approvers @djaglowski @jpkrohling
 connector/datadogconnector/                              @open-telemetry/collector-contrib-approvers @mx-psi @dineshg13
 connector/exceptionsconnector/                           @open-telemetry/collector-contrib-approvers @jpkrohling @marctc
-connector/failoverconnector/                             @open-telemetry/collector-contrib-approvers @akats7 @djaglowski @fatsheep9146
```

Comment:
@akats7 is the contributor of this component, maybe we should add them properly as code owner instead of removing here?

Reply:
This is reflecting what is in metadata.yaml. https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/172889d93b504c5180c879f2295d5a71fa84c3d7 doesn't list them in codeowners, worth a separate issue to follow up.

Reply:
That's fair. @akats7 can you open a PR adding yourself as code owner to this component?

Reply:
Thanks @crobert-1, yep will do